src/app/routes/utils/regex/utils/index.js

```diff
@@ -43,12 +43,12 @@ export const getManifestRegex = services => {
 
 export const getCpsAssetRegex = services => {
   const serviceRegex = getServiceRegex(services);
-  return `/:service(${serviceRegex})/:assetUri(${assetUriRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+  return `/:service(${serviceRegex}):variant(${variantRegex})?/:assetUri(${assetUriRegex}):amp(${ampRegex})?`;
 };
```

Comment:
This change is to ensure we have the variant in the correct place in the url for CPS assets: e.g. `https://www.bbc.com/serbian/cyr/svet-51656881`